database.yml

```diff
@@ -5,7 +5,7 @@ test:
   mysql2: &mysql2_defaults
     <<: *defaults
     username: root
-    prepared_statements: true
+    prepared_statements: false # See https://github.com/rails/rails/issues/53673
```

Comment:
The Rails team has resolved the issue in the meantine, but the changes are not released yet. Needs `> 8.0.0` for tests against Rails 8.